pkg/kubectl/resource_printer.go

```diff
@@ -716,6 +740,13 @@ func printPodList(podList *api.PodList, w io.Writer, options PrintOptions) error
 func printPodTemplate(pod *api.PodTemplate, w io.Writer, options PrintOptions) error {
 	name := pod.Name
 	namespace := pod.Namespace
+	kind := options.KindName
+
+	if options.WithKind {
+		if _, err := fmt.Fprintf(w, "%s\t", kind); err != nil {
```

Comment:
Is there a easy way to avoid putting this check in every kind of resource ? The uber function could first print the kind and then call these functions.